entity-types/ext-apache_mesos/definition.yml

```diff
@@ -0,0 +1,21 @@
+domain: EXT
+type: APACHE_MESOS
+synthesis:
+  rules:
+  - identifier: hostname
```

Comment:
Is this a stable globally unique identifier ?
Asking because telemetry of multiple apache mesos hosts may get mixed up on a single entity otherwise.

Also this makes me think if this entity type may be a 'APACHEMESOS_HOST' rather than a 'APACHE_MESOS' ?

Reply:
Yes, hostname is a stable identifier. We are using nri-flex on top of the infra agent to collect metrics. Should I change type from APACHE_MESOS to APACHEMESOS_HOST?

Reply:
I would discuss that with your product manager.
We are mostly checking for plausibility.
I think it depends on what the integration is actually instrumenting and what's a common wording in the Apache Mesos domain.

In general this makes sense to me.

Reply:
Sure, tagging @seemantk

Reply:
hi @andre-nr thank you.  When @sjyothi54 and I chatted earlier,  we felt like APACHEMESOS_CLUSTER is probably the correct entity to use here